### PR TITLE
fix for loading cases using elastic search

### PR DESCRIPTION
**Pull request by addison-jephtah**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/service/ccd-service.ts [hunk 1]
```diff
@@ -44,7 +44,8 @@ class CcdService {
       headers: {
         Authorization: headers.userToken,
         ServiceAuthorization: headers.serviceToken,
-        'content-type': 'application/json'
+        'content-type': 'application/json',
+        UserId: userId // Hack param to prove RIA-5761.
```
Comment (colmlynch60) [inline app/service/ccd-service.ts:48]: @hmcts-version1-ignacio @addison-jephtah @rajeshthuraiyur  - I added this userid header param which is only required to get tests running.

**Comment by colmlynch60** [comment]
@hmcts-version1-ignacio @addison-jephtah @rajeshthuraiyur  - 
 The change to get functional tests working can be seen in my two commits.
Previously, functional tests relied on user id to set up case data. The change to use elastic search removed the need for user id as searches are done using user token instead. 
To work around this - I added this userid header param which is only required to get tests running.
See loadCasesES.js that highlights the need for the change.

**Review by ivitale — APPROVED** [review]
lgtm

**Review by javimarinc96 — APPROVED** [review]
lgtm

**Pull request merged**